middleware.go

```diff
@@ -99,7 +99,7 @@ func SetupMiddleware(c config.ConfigReader, api *rest.Api) {
 	// For the rest of the requests expected Content-Type is 'application/json'.
 	api.Use(&rest.IfMiddleware{
 		Condition: func(r *rest.Request) bool {
-			if r.URL.Path == "/api/0.0.1/artifacts" && r.Method == http.MethodPost {
+			if r.URL.Path == "/api/management/v1/deployments/artifacts" && r.Method == http.MethodPost {
```

Comment:
Can you add a const in `routing.go` with this URL?

Reply:
thanks

Reply:
fixed, the new integration build will be available here:
https://mender-jenkins.mender.io/job/mender-builder/3143/